pyscraper/sp/get-official-reports-new.py

```diff
@@ -92,8 +109,13 @@ def main():
         for i in range(1, int(options.historic, 10)):
             get_document_from_id(i)
 
+    elif options.start_range and options.end_range:
+        for i in range(int(options.start_range), int(options.end_range, 10)):
```

Comment:
I realise that historic does the same, so perhaps it doesn't matter, just needs documenting better (historic too), I know I'd get confused at some point :)